### ISSUE TITLE
Error when filling questions of type time

### ISSUE THREAD

**Issue by freddieptf** [medic/cht-conf-test-harness#249]
**Describe the bug**
I can't successfully fill a time question using the test harness

**To Reproduce**

Form for reproducing [test.xlsx](https://github.com/medic/cht-conf-test-harness/files/14184719/test.xlsx)
```js
it('successfully submitted', async() => {
    const result = await harness.fillForm("test", [ "10:00"]);
    expect(result.errors).to.be.empty;
});

```
**What happened?**

```
Error: Evaluation failed: Error: Elements of type datetime expect input in format: "2022-12-31 13:21"
    at fillQuestion (webpack://cht-conf-test-harness/./src/form-host/form-filler.js?:176:15)
    at fillPage (webpack://cht-conf-test-harness/./src/form-host/form-filler.js?:110:5)
    at fillForm (webpack://cht-conf-test-harness/./src/form-host/form-filler.js?:59:26)
    at FormFiller.fillForm (webpack://cht-conf-test-harness/./src/form-host/form-filler.js?:30:42)
    at untransformedFillAndSave (webpack://cht-conf-test-harness/./src/form-host/index.js?:22:53)
    at window.fillAndSave (webpack://cht-conf-test-harness/./src/form-host/index.js?:53:78)
    at __puppeteer_evaluation_script__:1:36
    at ExecutionContext._evaluateInternal (node_modules/puppeteer-core/lib/cjs/puppeteer/common/ExecutionContext.js:221:19)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async ExecutionContext.evaluate (node_modules/puppeteer-core/lib/cjs/puppeteer/common/ExecutionContext.js:110:16)
    at async doFillPage (node_modules/cht-conf-test-harness/src/harness.js:760:22)
    at async Harness.fillForm (node_modules/cht-conf-test-harness/src/harness.js:316:24)
    at async Context.<anonymous> (test/forms/cebs_cha_signal_reporting_verification.spec.js:25:20)
```

**What did you expect?**
No errors

**Environment:**
- cht-conf 3.21.4
- cht-conf-test-harness 3.0.14

**Comment by jkuester** [comment]
This does seem to be a regression.  Filling out `time` questions works as expected for me on cht-conf-test-harness `2.4.3`. However, when I upgrade to `3.0.14`, I see this error.

**Comment by kennsippell** [comment]
Published as cht-conf-test-harness@3.0.15
